recirq/toric_code/toric_code_state_prep.py

```diff
@@ -0,0 +1,92 @@
+# Copyright 2022 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import annotations
+
+from typing import Iterator, List, Set
+
+import cirq
+
+from . import toric_code_rectangle as tcr
+
+
+class ToricCodeStatePrep:
```

Comment:
does this need to be a class? just use free functions

Reply:
Changed to free functions.